`libreoffice-qt` Fails to Build

### Describe the bug
When updating/installing `libreoffice-qt`, it fails with the following error:
```
configure: error: KF5 headers not found.  Please specify the root of your KF5 installation by exporting KF5DIR before running "configure".
error: builder for '/nix/store/nq2n6p077x4fhywaqij9lij7q5v80w3b-libreoffice-7.2.5.2.drv' failed with exit code 1;
       last 10 log lines:
       > checking for QtCore/qconfig.h... yes
       > checking for Qt5 libraries... /nix/store/g6lhjbkcz9nbwlsdd83qsa3rg6i90rs7-qtbase-5.15.3/lib
       > checking for xcb... yes
       > checking for xcb-icccm... yes
       > checking for moc-qt5... no
       > checking for moc... /nix/store/0h41dvl7ipzzm2nj3qfd3vkvaiw6iz9z-qtbase-5.15.3-dev/bin/moc
       > checking for gobject-2.0... yes
       > checking for kf5-config... no
       > checking for KF5 headers... no
       > configure: error: KF5 headers not found.  Please specify the root of your KF5 installation by exporting KF5DIR before running "configure".
       For full logs, run 'nix log /nix/store/nq2n6p077x4fhywaqij9lij7q5v80w3b-libreoffice-7.2.5.2.drv'.
error: 1 dependencies of derivation '/nix/store/7k73impmw6li61ryjn91wix5sjsln2im-libreoffice-7.2.5.2.drv' failed to build
```

### Steps To Reproduce
Steps to reproduce the behavior:
1. Execute `nix-shell -p libreoffice-qt`
2. Observe build failure

### Expected behavior
libreoffice-qt to install without errors.

### Additional context
KDE Plasma 5.24.0 just landed in `master`, so this might be related to it.

### Notify maintainers
@7c6f434c

### Metadata
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.29, NixOS, 22.05 (Quokka)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.8.0pre20220311_d532269`
 - channels(root): `"nixos"`
 - channels(bill): `"home-manager"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

--- a/pkgs/applications/office/libreoffice/default.nix
+++ b/pkgs/applications/office/libreoffice/default.nix
@@ -110,6 +110,7 @@ in (mkDrv rec {
       'GPGMEPP_CFLAGS=-I${gpgme.dev}/include/gpgme++'
   '' + lib.optionalString kdeIntegration ''
       substituteInPlace configure.ac \
+        --replace kcoreaddons_version.h KCoreAddons/kcoreaddons_version.h \
         --replace '$QT5INC'             ${qtbase.dev}/include \
         --replace '$QT5LIB'             ${qtbase.out}/lib \
         --replace '-I$qt5_incdir '      '-I${qtx11extras.dev}/include '\

